### PR TITLE
docs: mention usage of management commands

### PR DESCRIPTION
**Pull request by Ali-Salman29**
**Description**

Added usage of the following management commands in ReadMe.

- Creating MongoDB indexes
- Initialize search indices
- Rebuild search indices

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -127,6 +127,40 @@ To try out changes before applying them, use the ``--dry-run`` option. For insta
 
    ./manage.py lms forum_delete_course_from_mongodb all --dry-run
 
+Building MongoDB Indexes
+------------------------
+
+To optimize MongoDB query performance, it is crucial to create database indexes. The command will create or update indexes and skip them if they already exist.
+
+To create or update MongoDB indexes, execute the following command:
```
Comment (regisb) [inline README.rst:135]: Commands must be prefixed by "::"

    To create or update MongoDB indexes, execute the following command::

(same below)

File: README.rst [hunk 2]
```diff
@@ -127,6 +127,40 @@ To try out changes before applying them, use the ``--dry-run`` option. For insta
...
+---------------
+
+Based on your search backend i.e Elasticsearch or Meilisearch, the commands will populate search indexes.
+
+Initialize Forum Indices
+-------------------------
+
+The `initialize_forum_indices` command used to initialize Elasticsearch indices for the forum. It allows you to force the creation of new indices even if they already exist.
+
+    ./manage.py lms initialize_forum_indicies
```
Comment (regisb) [inline README.rst:151]: initialize_forum_indicies -> initialize_forum_indices

File: README.rst [hunk 3]
```diff
@@ -127,6 +127,40 @@ To try out changes before applying them, use the ``--dry-run`` option. For insta
...
+
+Search indicies
+---------------
+
+Based on your search backend i.e Elasticsearch or Meilisearch, the commands will populate search indexes.
+
+Initialize Forum Indices
+-------------------------
+
+The `initialize_forum_indices` command used to initialize Elasticsearch indices for the forum. It allows you to force the creation of new indices even if they already exist.
```
Comment (regisb) [inline README.rst:147]: Verbatim code is surrounded in double backticks:

    The ``initialize_forum_indices`` command...

(same below)

File: README.rst [hunk 4]
```diff
@@ -127,6 +127,40 @@ To try out changes before applying them, use the ``--dry-run`` option. For insta
...
+Building MongoDB Indexes
+------------------------
+
+To optimize MongoDB query performance, it is crucial to create database indexes. The command will create or update indexes and skip them if they already exist.
+
+To create or update MongoDB indexes, execute the following command:
+
+    ./manage.py lms forum_create_mongodb_indexes
+
+Search indicies
```
Comment (regisb) [inline README.rst:139]: indicies -> indices

But this section is not useful, we should remove it.

File: README.rst [hunk 5]
```diff
@@ -127,6 +127,40 @@ To try out changes before applying them, use the ``--dry-run`` option. For insta
...
+The `initialize_forum_indices` command used to initialize Elasticsearch indices for the forum. It allows you to force the creation of new indices even if they already exist.
+
+    ./manage.py lms initialize_forum_indicies
+
+Pass `--force` flag to force the creation of new indices.
+
+Rebuild Forum Indices
+----------------------
+
+The `rebuild_forum_indices` command is used to rebuild Elasticsearch indices from scratch. This command allows you to specify the batch size for processing documents and extra minutes for catch-up.
```
Comment (regisb) [inline README.rst:158]: ...to rebuild search indices...

File: README.rst [hunk 6]
```diff
@@ -127,6 +127,40 @@ To try out changes before applying them, use the ``--dry-run`` option. For insta
...
+
+To run the command, use the following syntax:
+
+    ./manage.py lms rebuild_forum_indices --batch_size <size> --extra_catchup_minutes <minutes>
```
Comment (regisb) [inline README.rst:162]: Let's not mention the batch_size and extra_catchup_minutes options. They are not necessary, since they have good defaults.

**Review by regisb — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (Ali-Salman29) [reply README.rst]: Yes, I'm updating form the GitHub UI.

**Review by regisb — APPROVED** [review]
:+1:

**Pull request merged**